### PR TITLE
WIP Blog Tag Update

### PR DESCRIPTION
**Pull request by CodingItWrong**
## What it does
Adds some new tags to the blog and sidebar to better organize recent posts, and removes a few little-used tags from the sidebar.

image

### CODE REVIEW

**Comment by kennethlarsen** [comment]
I have a hard time grasping the `Tech` category. What exactly goes into `Tech`?

File: source/blog/_sidebar.html.erb [hunk 1]
```diff
@@ -65,13 +65,12 @@
   <h3 class="list-group-title" id="browse-subject">Browse by Subject</h3>
   <ul class="list-group" aria-labelledby="browse-subject">
     <li class="list-item"><a href="<%= tag_path 'Announcement'%>">Announcements</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Ember CLI'%>">Ember CLI</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Community'%>">Community</a></li>
     <li class="list-item"><a href="<%= tag_path 'Ember Data'%>">Ember Data</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Performance'%>">Performance</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Leadership'%>">Leadership</a></li>
```
Comment (jessica-jordan) [inline source/blog/_sidebar.html.erb]: What do you think about renaming the tag `Leadership` to `Steering Committee` or something similar? I think this aligns even better with the new wordings from the Ember Community Reorganization announcement:

https://www.emberjs.com/blog/2018/06/30/organizing-our-contributors.html

File: source/blog/_sidebar.html.erb [hunk 2]
```diff
@@ -65,13 +65,12 @@
...
     <li class="list-item"><a href="<%= tag_path 'Roadmap'%>">Roadmap</a></li>
     <li class="list-item"><a href="<%= tag_path 'Security'%>">Security</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Tech'%>">Tech</a></li>
```
Comment (jessica-jordan) [inline source/blog/_sidebar.html.erb]: I'm _assuming_ that most posts that fall under the category "Tech" are already covered by either the `Releases` or the `Ember CLI` / `Ember Data` tags. What are your thoughts?

**Comment by sivakumar-kailasam** [comment]
Closed it originally with the intention to reopen it to account for a misconfiguration on Travis. But couldn't because the fork no longer exists.

**Comment by CodingItWrong** [comment]
Yup, I accidentally deleted my fork. I've lost momentum on the topic; I'm not passionate enough about the tagging to drive conversation toward consensus. Maybe when the time is right someone else can pick up where I left off!

**Pull request closed without merging**